sspl_test/coverage/coverage_setup.py

```diff
@@ -57,31 +57,25 @@ def generate_cov_report(signal_number, frame):
         outfile='%scoverage/sspl_xml_coverage_report.xml')
     logger.info(f'XML coverage report generated with coverage of {cov_per} percentage.')
     ## Enable below code to inable HTML report generation
- 	# html_cov_per = co.html_report(
- 	#                     directory='%scoverage/sspl_html_coverage',
- 	#                     ignore_errors=True,
- 	#                 )
- 	# logger.info(f'HTML coverage report geverated with coverage of {html_cov_per} percentage.')\
-"""%(DATA_PATH, DATA_PATH)
+    # html_cov_per = co.html_report(
+    #                     directory='%scoverage/sspl_html_coverage',
+    #                     ignore_errors=True,
+    #                 )
+    # logger.info(f'HTML coverage report geverated with coverage of {html_cov_per} percentage.')\
+""" % (DATA_PATH, DATA_PATH)
 
 PATCH_4 = """\
     signal.signal(signal.SIGUSR1, generate_cov_report)\
 """
 
+
 def coverage_setup():
-    """Installs pip3 coverage pkg and creates required files for code coverage.
+    """Creates required files for code coverage.
```

Comment:
```suggestion
    """
    Creates required files for code coverage.
```

Reply:
addressed, Now throws codacy issue to start at first line 🤣

Reply:
Yes, we need to get one of them disabled.

Reply:
Some codacy issues keep conflicting